packages/service-library/src/servicelib/rabbitmq/_utils.py

```diff
@@ -55,11 +55,12 @@ async def declare_queue(
     exchange_name: str,
     *,
     exclusive_queue: bool,
+    message_ttl: int = _RABBIT_QUEUE_MESSAGE_DEFAULT_TTL_MS,
```

Comment:
question: how come your linter accepts this? you call this with ```int | None``` and here it is an ```int```?

Reply:
It did not accept it, all tests were failing :) it was just Friday evening commit that I didn't properly check, it is already fixed now